### PR TITLE
Add how to not run local instance of MongoDB to README.md

### PR DESCRIPTION
**Pull request by JohnAllen**
This is buried in a closed issue so thought it would be helpful to have it in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -277,6 +278,24 @@ You can't access the MongoDB from outside the server. To access the MongoDB shel
...
+//  oplog: true,
+//  port: 27017,
+//  version: '3.4.1' // (optional), default is 3.4.1
+//  servers: {
+//  one: {},
+//    },
+//  },
+```
+You'll also need to change your `MONGO_URL` to point to your external Mongo
+instsance(s): `MONGO_URL: 'mongodb://some.other.ip.address:27017/dbname'`
```
Comment (zodern) [inline README.md:297]: Typo

File: README.md [hunk 2]
```diff
@@ -277,6 +278,24 @@ You can't access the MongoDB from outside the server. To access the MongoDB shel
 
 > Later on we'll be using a separate MongoDB instance for every app.
 
+### Using External Mongo Instances
+
+To not have a local docker instance of Mongo installed and run, simply comment
+out the `mongo` section in your `mup.js` config file:
```
Comment (zodern) [inline README.md:284]: > simply comment out

Please change it to "simply remove or comment out"

Also, I'm not sure the example is needed. What is your opinion?

File: README.md [hunk 3]
```diff
@@ -26,6 +26,7 @@ This version of Meteor Up is powered by [Docker](http://www.docker.com/), making
     - [Deploy Wait Time](#deploy-wait-time)
     - [Multiple Deployment Targets](#multiple-deployment-targets)
 - [Accessing the Database](#accessing-the-database)
+- [Using External Mongo Instances](#accessing-the-database)
```
Comment (zodern) [inline README.md:29]: It links to the wrong section.

**Review by zodern — CHANGES_REQUESTED** [review]
Thanks. This is needed.

A couple things you could add:
- When you might want to use an external mongo instance. (Oplog, multiple servers running app, etc)
- List of companies offering MongoDB DBaaS

File: README.md (hunk 1 above)
Comment (shadowcodex) [reply README.md:297]: Denote that `MONGO_URL` is under the Meteor.ENV object.

File: README.md (hunk 2 above)
Comment (shadowcodex) [reply README.md:284]: Example is not needed IMO.

File: README.md (hunk 3 above)
Comment (JohnAllen) [reply README.md:29]: Yeah I was doing this hastily as evidenced by the mistakes.

**Comment by shadowcodex** [comment]
@JohnAllen are you able to make the requested changes?

**Comment by JohnAllen** [comment]
Yes sure give me 36 hours. Been traveling.

**Pull request closed without merging**